pepper-apis/studybuilder-cli/tenants/cmi/tenant.yaml

```diff
@@ -18,6 +18,39 @@ rules:
     enabled: true
     order: 2
 
+actions:
+  - name: Check Admin Role
+    code: ../actions/check-admin-role.js
+    deployed: false
+    status: built
+    supported_triggers:
+      - id: post-login
+        version: v1
+
+  - name: Register User in Pepper
+    code: ../register-user-in-pepper-action.js
+    dependencies:
+      - name: request
+        version: 2.88.2
+      - name: google-cloud/logging
```

Comment:
Are we logging from auth0 actions to GCP logging?  Can you say a bit more about the dependency?

Reply:
nvm, I see it below...